db/migrate/20220520160742_remove_financial_officers_if_more_then_one.rb

```diff
@@ -0,0 +1,17 @@
+class RemoveFinancialOfficersIfMoreThenOne < ActiveRecord::Migration[6.0]
```

Comment:
As far as i understand it's generally not best practice to do data migration in db schema migrations.

Sources: [here](https://www.ombulabs.com/blog/rails/data-migrations/three-useful-data-migrations-patterns-in-rails.html) and [here](https://thoughtbot.com/blog/data-migrations-in-rails)